Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "clam"
-version = "0.2.10"
```

Comment:
I just realized, we don't need to update the rust toml with pyclam, as the python code has no way of being included in the eventual rust crate. We could reset this to 0.1.0 and tie bumps to this to correspond with actual rust releases. The version numbers would be misaligned, but I think that's more informative then this linking we've done so far.

Reply:
Basically, I'd argue the python and rust implementations *should* be on different version numbers to indicate the relative progress both packages have had

Reply:
I agree with this. Let's start this with the CHESS branch.